src/org/opendatakit/briefcase/model/form/FileSystemFormMetadataAdapter.java

```diff
@@ -132,12 +133,23 @@ private static FormMetadata deserialize(Path storageRoot, Path metadataFile) {
 
   private static Path serialize(FormMetadata metaData) {
     try {
-      return write(
-          getMetadataFile(metaData),
-          MAPPER.writerWithDefaultPrettyPrinter().writeValueAsString(metaData.asJson(MAPPER)),
-          CREATE, TRUNCATE_EXISTING
-      );
-    } catch (JsonProcessingException e) {
+      Path formDir = metaData.getFormDir();
```

Comment:
This method has duplicated code that could be simplified by calling `UncheckedFiles.createDirectories()` unconditionally before the `write` call.

Reply:
That is correct. I will make the changes right now.